### PR TITLE
Ci python 38

### PR DESCRIPTION
**Pull request by mfeurer**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -72,6 +72,13 @@ jobs:
     - name: Install scikit-learn ${{ matrix.scikit-learn }}
       run: |
         pip install scikit-learn==${{ matrix.scikit-learn }}
+    - name: Install numpy for Python 3.8
+      # Python 3.8 & scikit-learn<0.24 requires numpy<=1.23.5
+      if: ${{ matrix.python-version == '3.8' && contains(fromJSON('["0.23.1", "0.22.2", "0.21.2"]'), matrix.scikit-learn) }}
+      run: |
+        echo ${{ matrix.python-version }} == '3.8'
+        echo ${{ matrix.python-version }}
```
[the remark this answers is not in the document]
Comment (mfeurer) [reply .github/workflows/test.yml:80]: No, for me for debugging. I will remove them (and also fix codecov) in the next push.

**Comment by PGijsbers** [comment]
Thought I had a review request so I had a look. Progress so far looks good to me 👍

**Comment by mfeurer** [comment]
Unfortunately, I don't know how to fix the coverage. If you have any idea, that would be appreciated. Otherwise, this is good to merge as it improves our CI quite a bit.

**Review by PGijsbers — APPROVED** [review]
Let's look into that in a separate PR.

**Pull request merged**